portfolio/src/main/webapp/script.js

```diff
@@ -31,12 +31,6 @@ function getComments(){
     });
 }
 
-function comments_to_display(){
-    let num = document.getElementById("num_of_comments");
-    let number = num.options[num.selectedIndex].value;
-    console.log(number);
-}
-
```

Comment:
Deleting this seems unrelated to the map. Ditto the function `addRandomGreeting`. If these functions were unused, I guess it's fine to delete them in this PR. But please mention in the PR description that you're also deleting some unused code.

Reply:
Okay, these were functions I just didn't need anymore. I'll be sure to make note of them next time